Cultiv.Hangfire/HangfireComposer.cs

```diff
@@ -64,7 +64,7 @@ private static void AddAuthorizedUmbracoDashboard(IUmbracoBuilder builder)
                 {
                     endpoints.MapHangfireDashboardWithAuthorizationPolicy(
                         pattern: Constants.System.Endpoint,
-                        options: new DashboardOptions(),
+                        options: new DashboardOptions() { IgnoreAntiforgeryToken = true },
```

Comment:
```suggestion
                        options: new DashboardOptions { IgnoreAntiforgeryToken = true },
```

Don't need the parentheses here any more! 😁